plugin-dev/Source/Sentry/Sentry_Android_UPL.xml

```diff
@@ -134,22 +152,30 @@
         </insert>
     </baseBuildGradleAdditions>
 
-    <buildscriptGradleAdditions>
-        <insert>
-            dependencies {
-                classpath 'com.android.tools.build:gradle:3.5.4'
-            }
-        </insert>
-    </buildscriptGradleAdditions>
-
     <buildscriptGradleAdditions>
         <if condition="bUploadSymbols">
             <true>
-                <insert>
-                    dependencies {
-                        classpath 'io.sentry:sentry-android-gradle-plugin:2.1.5'
-                    }
-                </insert>
+                <if condition="bUseLegacyGradlePlugin">
+                    <true>
+                        <log text="Using legacy Sentry Gradle plugin 2.1.5 for Android debug symbol upload."/>
+                        <insert>
+                            dependencies {
+                                classpath 'com.android.tools.build:gradle:3.5.4'
+                                classpath 'io.sentry:sentry-android-gradle-plugin:2.1.5'
+                            }
+                        </insert>
+                    </true>
+                    <false>
+                        <log text="Using Sentry Gradle plugin 4.11.0 for Android debug symbol upload."/>
```

Comment:
Is the hardcoded version number going to bite us in the butt later on?

Reply:
Since we don't have any means allowing to check how Sentry Gradle plugin performs in our CI probably it'd be safer to keep a hardcoded version here instead of bumping it automatically for the time being.